packages/ui/src/components/primitives/base/Icons.tsx

```diff
@@ -176,3 +176,22 @@ export const CloseIcon = () => {
 		</svg>
 	);
 };
+
+export const LockIcon = ({ className }: { className?: string }) => {
```

Comment:
We have headless icons and bootstrap icons installed, both of those have a lock component, do we need to have this?

Reply:
Fixed